spec/index.html

```diff
@@ -371,26 +408,66 @@ <h3>Triple Terms and Reification</h3>
-        The diagram represents the proposition as a fact using the asserted triple, meaning that the relationship holds.
+<!-- 
+Slightly rewriting the below statement.        
+-->
+<!-- original:
+The diagram represents the proposition as a fact using the asserted triple, meaning that the relationship holds.
+-->
+        Due to the asserted triple, the diagram represents the proposition as a fact, meaning that the relationship holds.
       </figcaption>
     </figure>
 
+<!-- I found the sentence to be quite difficult to read - here is an attempt to simplify. -->
+<!-- original:
+    <p>When the triple term of a <a>reifying triple</a> also [=appears=] in the
+      same graph as an <a>asserted triple</a>, the subset of triples that share
+      the same <a>reifier</a> as subject is called a <dfn>triple annotation</dfn>.</p>
+-->
+
+    <p>Here, the triple term of a <a>reifying triple</a> also [=appears=] in the
+      same graph as an <a>asserted triple</a>. In this case, 
+      the subset of triples including the <a>reifier</a> as subject&mdash;for instance, providing metadata as in these examples&mdash;
+      is called a <dfn>triple annotation</dfn>.
+    </p>
+
+    <p>Finally, [=RDF terms=] that [=appear=] in a [=triple term=] have the same
+      <a data-lt="denotes">denotation</a> as when they appear in an
+      [=asserted triple=] in the <a data-lt="RDF graph">graph</a>.
+<!-- Adding an example here -->
+For instance, the term <code>:Alice</code> from a [=triple term=] and <code>:Alice</code> in an [=asserted triple=] both denote the same resource.
+      For this reason, we say that triple terms are
+      <dfn class="lint-ignore">transparent</dfn>.</p>
+
+    <p>
+      Concrete syntaxes, such as Turtle [[RDF12-TURTLE]], may have shortcuts
+      for specifying <a>reifying triples</a> and <a>triple annotations</a> more
+      succinctly.
+    </p>
+
     <div class="note">
+<!-- RDF triples (which triple terms are) are said to encode propositions, not denote them. AFAIK, terms in RDF denote resources. -->
       <p><a>Triple terms</a> always denote abstract, logical <a>propositions</a>,
         while <a>reifiers</a> may denote a variety of things that are related
         to these propositions (such as a statement or belief that the
         proposition holds, or an event or circumstance that makes the
-        proposition true).
+        proposition true). 
+<!-- Could an example be given here? -->
         It is therefore expected that triple terms are commonly used as
         <a>objects</a> of <a>reifying triples</a>, and that the reifiers
-        (rather than the triple terms) are used in further descriptions.</p>
+        (rather than the triple terms) are used in further descriptions.
+<!-- Adding this part to avoid confusion (we're not talking about anything new) -->
+        This section describes this common usage.
```

Comment:
"section" is a little confusing inside a note box.

Does it mean "the note box" or document 1.5?

Reply:
I indeed meant the section in which the note box is located, so, Section 1.5.

Reply:
Added a reference to Section 1.5 explicitly. (Apologies if there's a better way to do internal referencing in this document; feel free to do it differently.)